lighthouse-core/report/html/templates.html

```diff
@@ -783,24 +783,63 @@
 
 <template id="tmpl-lh-3p-filter">
   <style>
+    /* The input checkbox element is invisible but should span the same width and height as the label. */
     .lh-3p-filter {
-      background-color: var(--table-higlight-background-color);
       color: var(--color-gray-600);
       float: right;
       padding: 6px;
+      position: relative;
     }
-    .lh-3p-filter-label, .lh-3p-filter-input {
-      vertical-align: middle;
-      user-select: none;
+    .lh-3p-filter__input {
+      cursor: pointer;
+      opacity: 0;
+      width: 100%;
```

Comment:
width/height 100 makes the input take the width/height of the container, encompassing the label

![image](https://user-images.githubusercontent.com/4071474/63469750-cf274b80-c41f-11e9-8655-2ea6ab545cc9.png)

otherwise it is 13x13

IDK if this is important ... but the metric pill does this so I copied the behavior.

Reply:
oh it's not `display: block` :) ya ya sounds good 👍